app/bundles/PluginBundle/Resources/views/Integration/info.html.twig

```diff
@@ -16,7 +16,7 @@
 {% if bundle.hasSecondaryDescription %}
 <div class="row mt-lg">
     <div class="col-xs-12">
-        {{ bundle.secondaryDescription|raw }}
```

Comment:
@escopecz I think it is a good idea to check where we have `raw` in the whole code. it is a potential bug

Reply:
Good point. I went through the usages of `|raw` and found some that can be purifyied in https://github.com/mautic/mautic/pull/12528/commits/6a92d8675d9190c46c0efef69f7b36698bab8141